CONTRIBUTING.md

```diff
@@ -0,0 +1,146 @@
+# CONTRIBUTING
+
+## Introduction
+
+Hello and thank you for your interest in contributing to Log Viewer.
+
+Contributions are welcome and there are many ways you can get involved!
+
+To get started, choose your area of interest:
+
+<table>
+  <tr>
+    <td  align="center">
+        <a href="#-issues--discussions">👥 Issues & Discussions</a> |
+        <a href="#-documentation">📚 Documentation</a> |
+        <a href="#-spread-the-word">📣 Spread the word</a> |
+        <a href="#-code-contribution">💻 Code Contribution</a>
+    </td>
+  </tr>
+</table>
+
+<br/>
+
+---
+
+### 👥 Issues & Discussions
+
+You can interact with users by sharing information and asking/answering questions in our [Discussions](https://github.com/opcodesio/log-viewer/discussions) tab.
+
+Also, you can contribute by reporting bugs, patching problems or providing technical support in our [Issues](https://github.com/opcodesio/log-viewer/issues) tab.
+
+<br/>
+
+---
+
+### 📚 Documentation
+
+Documentation is key for any project success!
+
+Currently, our documentation is stored at the [README](https://github.com/opcodesio/log-viewer/blob/main/README.md) file of this repository.
+
+You may contribute by improving existing information, covering missing topics, or fixing typos and grammar errors.
+
+The documentation official language is in English.
+
+<br/>
+
+---
+
+### 📣 Spread the word
+
+If you enjoy Log Viewer, please consider talking about our project in your community.
+
+Share this [repository link](https://github.com/opcodesio/log-viewer/) on Twitter, YouTube, Discord or any other social network you are part of.
+
+You are also welcome to write articles, reviews and tutorials about this project on your blog or programming website.
+
+Ah! Don't forget to let the author know about your work. Say hello to [@arukompas](https://github.com/arukompas).
+
+<br/>
+
+---
+
+### 💻 Code Contribution
+
+Please follow the steps below to contribute with code.
+
+## Steps
+
+### 📌 Step 1
+
+Fork this repository and enter its directory. Run the command:
+
+```shell
+git clone https://github.com/opcodesio/log-viewer.git && cd log-viewer
+```
```

Comment:
This might be a little confusing. It asks to fork the repo, but then clone the official repo. They won't be able to push changes or branches into the official repo and will possibly get stuck later on.

I suggest leaving a placeholder instead of `opcodesio` username, so that it's clear it should be their own, forked repo:

```
git clone https://github.com/<YOUR-USERNAME>/log-viewer.git && cd log-viewer
```